.gitignore

```diff
@@ -26,7 +26,11 @@
 !/tmp/storage/.keep
 
 /public/assets
-
+/public/packs
+/public/packs-test
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221382911-017cf4e1-a1c0-4b61-82d1-1acc8f91311d.png)

**[Optional]**

- [ ] It would be nice if you remove this Live Demo section since you don't have a live link for your project :thinking: Kindly remove it now to make your readme file descriptive and professional :+1: Once you will have your live link ready then you can add this Live Demo section with the live link in the readme file :blush:

Reply:
@HAMMAS-SALEEM, Alright, I will. Thank you so much. I appreciate it a lot. 🤗